wolk/wolkabout_protocol_message_deserializer.py

```diff
@@ -34,75 +33,97 @@ class WolkAboutProtocolMessageDeserializer(MessageDeserializer):
     :vartype logger: logging.Logger
     """
 
-    DEVICE_PATH_DELIMITER = "d/"
-    REFERENCE_PATH_PREFIX = "r/"
+    PLATFORM_TO_DEVICE = "p2d/"
     CHANNEL_DELIMITER = "/"
-    KEEP_ALIVE_RESPONSE = "pong/"
-    ACTUATOR_SET = "p2d/actuator_set/"
-    CONFIGURATION_SET = "p2d/configuration_set/"
-    FILE_BINARY_RESPONSE = "p2d/file_binary_response/"
-    FILE_DELETE = "p2d/file_delete/"
-    FILE_PURGE = "p2d/file_purge/"
-    FILE_LIST_CONFIRM = "p2d/file_list_confirm/"
-    FILE_LIST_REQUEST = "p2d/file_list_request/"
-    FILE_UPLOAD_ABORT = "p2d/file_upload_abort/"
-    FILE_UPLOAD_INITIATE = "p2d/file_upload_initiate/"
-    FILE_URL_DOWNLOAD_ABORT = "p2d/file_url_download_abort/"
-    FILE_URL_DOWNLOAD_INITIATE = "p2d/file_url_download_initiate/"
-    FIRMWARE_UPDATE_ABORT = "p2d/firmware_update_abort/"
-    FIRMWARE_UPDATE_INSTALL = "p2d/firmware_update_install/"
+
+    TIME = "time"
```

Comment:
You have this parameters in two moduels defined. Export it into separated module.